LeetCodeSolutions/Program.cs

```diff
@@ -1,3 +1,35 @@
-﻿// See https://aka.ms/new-console-template for more information
-
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+﻿namespace LeetCodeSolutions;
+public class Program
+{
+    public bool IsValid(string s) {
+        Stack<char> strStack = new Stack<char>();
+        Dictionary<char, char> parentheses = new Dictionary<char, char>
+        {
+            { '(', ')' },
+            { '{', '}' },
+            { '[', ']' }
+        };
+        foreach(var p in s){
+            if (parentheses.ContainsKey(p)) strStack.Push(p);
```

Comment:
Can't we assume that the character p would always be one of the keys for _parentheses_ based on the constraints of the question?